### PR TITLE
On-prem REDIRECT with NPT

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerNetworking.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    /** Each container has an associated macvlan bridge. */
+    MACVLAN,
+
+    /** Network Prefix-Translated networking. */
+    NPT,
+
+    /** A host running a single container in the host network namespace. */
+    HOST_NETWORK;
+
+    public static DockerNetworking from(String cloud, NodeType nodeType, boolean hostAdmin) {
```
Comment (freva) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerNetworking.java]: This should just be set in hosted code when creating the `Environment`?

**Review by freva — APPROVED** [review]

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerNetworking.java (hunk 1 above)
Comment (hakonhall) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/docker/DockerNetworking.java]: Possibly

**Comment by hakonhall** [comment]
Btw, @smorgrav the REDIRECT is supposed to help with what? From the documentation, it seemed like this REDIRECT is for communicating with oneself, inside the container. However that worked just fine without this PR.

**Comment by smorgrav** [comment]
If you talk to yourself on an IP that is otherwise routed to the bridge. I.e using your 'public' IP that does not belong the the internal interface

**Pull request merged**